addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -396,14 +396,13 @@
             <img t-attf-src="/web/image/gamification.karma.rank/#{user.rank_id.id}/image_small" height="16" alt="" class="mr-1"/>
             <a href="/profile/ranks_badges" t-field="user.rank_id"/>
         </div>
-        <t t-if="user.rank_id">
-            <div t-if="user.next_rank_id" class="font-weight-bold text-muted mt-1">Next rank:</div>
-            <t t-call="website_profile.profile_next_rank_card">
-                <t t-set="bg_class">bg-200</t>
-                <t t-set="img_max_width">50%</t>
-            </t>
-            <div t-field="user.rank_id.description_motivational"/>
+        <t t-set="next_rank_id" t-value="user._get_next_rank()"/>
+        <div t-if="next_rank_id" class="font-weight-bold text-muted mt-1">Next rank:</div>
+        <t t-call="website_profile.profile_next_rank_card">
+            <t t-set="bg_class">bg-200</t>
+            <t t-set="img_max_width">50%</t>
         </t>
+        <div t-if="next_rank_id" t-field="next_rank_id.description_motivational"/>
```

Comment:
We talked yesterday about making this:

```
<t t-if="next_rank_id">
    <div t-field="next_rank_id.description_motivational"/>
</t>
<t t-else="">
    <div>Congratulations, you have reached the last rank!</div>
</t>
```

Could be good for layout consistency and user e-penis but I let you decide whether to change it or not ^^

Reply:
I like e-penis.